app/controllers/course/assessment/submission_question/comments_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+class Course::Assessment::SubmissionQuestion::CommentsController < Course::Assessment::Controller
```

Comment:
Would it be better to have this under the Submission namespace instead - `Course::Assessment::Submission::SubmissionQuestion`